src/System.Diagnostics.TraceSource/pkg/System.Diagnostics.TraceSource.pkgproj

```diff
@@ -3,13 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Diagnostics.TraceSource.csproj">
-      <SupportedFramework>net46;netstandardapp1.5</SupportedFramework>
+      <SupportedFramework>net46;netcore50;netstandardapp1.5</SupportedFramework>
```

Comment:
Are we missing a `<NotSupportedOnTargetFramework Include="netcore50"/>`?  This is another one of those contracts that uses PInvokes that violate WACK.